wiki/Tournaments/OWC/2013/de.md

```diff
@@ -1,122 +1,122 @@
+| ![][flag_VE] | Venezuela | **[MeowinTurtle](https://osu.ppy.sh/users/2026980)**, Baozis <!-- fehlt -->, [CrymynaL](https://osu.ppy.sh/users/1158908), [Livean](https://osu.ppy.sh/users/674036), [Roli](https://osu.ppy.sh/users/1797688), [S4suk3](https://osu.ppy.sh/users/401955) |
+
+## Gruppen
+
+| Top Seed | High Seed | Mid Seed | Low Seed |
+| :-- | :-- | :-- | :-- |
+| ![][flag_CN] China | ![][flag_AR] Argentinien | ![][flag_AU] Australien | ![][flag_BE] Belgien |
+| ![][flag_DE] Deutschland | ![][flag_BR] Brasilien | ![][flag_AT] Österreich | ![][flag_BG] Bulgarien |
+| ![][flag_JP] Japan | ![][flag_HK] Hong Kong | ![][flag_CA] Kanada | ![][flag_CL] Chile |
+| ![][flag_KR] Südkorea | ![][flag_LV] Lettland | ![][flag_FI] Finnland | ![][flag_EE] Estland |
+| ![][flag_PL] Poland | ![][flag_NO] Norway | ![][flag_FR] France | ![][flag_NZ] New Zealand |
+| ![][flag_TW] Taiwan | ![][flag_RU] Russische Föderation | ![][flag_ID] Indonesien | ![][flag_PH] Philippinen |
+| ![][flag_TH] Thailand | ![][flag_SE] Schweden | ![][flag_IT] Italien | ![][flag_SG] Singapur |
+| ![][flag_US] Vereinigte Staaten | ![][flag_GB] Vereinigtes Königreich | ![][flag_NL] Niederlande | ![][flag_VE] Venezuela |
+
+![](img/bracket.jpg "OWC 2013 Turnierbaum")
+
+## Podium
+
+Dieser Wettbewerb ist zu Ende gegangen und hat die folgenden Podiumsplätze ergeben:
+
+| Platzierung | Team |
+| :-: | :-- |
+| ![Goldkrone](/wiki/shared/crown-gold.png "1st place") | ![][flag_KR] **Südkorea** (**[Dungeon](https://osu.ppy.sh/users/461720)**, [- Hakurei Reimu-](https://osu.ppy.sh/users/948713), [CheEZ](https://osu.ppy.sh/users/272117), [dragonhuman](https://osu.ppy.sh/users/713266), [ffury](https://osu.ppy.sh/users/2056071), [K i R i K a R u](https://osu.ppy.sh/users/139670), [Shizuru-](https://osu.ppy.sh/users/1341421), [Tengu](https://osu.ppy.sh/users/380836)) |
+| ![Silverkrone](/wiki/shared/crown-silver.png "2nd place") | ![][flag_TW] **Taiwan** (**[Uan](https://osu.ppy.sh/users/147623)**, [dabanlong](https://osu.ppy.sh/users/624254), [I will be back](https://osu.ppy.sh/users/477704), [onlyforyou](https://osu.ppy.sh/users/597858), [Rucker](https://osu.ppy.sh/users/147515), [Small K](https://osu.ppy.sh/users/952751), [SnowWhite](https://osu.ppy.sh/users/50265), [YuyuKo sama](https://osu.ppy.sh/users/234788)) |
+| ![Bronzekrone](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_PL] **Polen** (**[fartownik](https://osu.ppy.sh/users/56917)**, [AmaiHachimitsu](https://osu.ppy.sh/users/844815), [Beniek](https://osu.ppy.sh/users/1649633), [Niko](https://osu.ppy.sh/users/175141), [Piotrekol](https://osu.ppy.sh/users/304520), [rEdo](https://osu.ppy.sh/users/49329), [worst fl player](http://osu.ppy.sh/users/613592), [WubWoofWolf](https://osu.ppy.sh/users/39828)) |
 
 ## Mappools
```

Comment:
A bit unsure if the usage of *beatmap* instead of *map* is better here 🤔 

This would lead to translations such as `Beatmap-Pools` and `Beatmappack`/`Beatmap-Paket` (ignoring the dash writing style). Putting this here for further discussion.

Reply:
Then let's go with this, as you've suggested:
- mappool --> Mappool
- mappack --> Mappack
- map --> Beatmap

👍